core/src/com/cloud/storage/template/VhdProcessor.java

```diff
@@ -91,27 +97,58 @@ public long getVirtualSize(File file) throws IOException {
         return file.length();
     }
 
-    protected long getTemplateVirtualSize(File file) throws IOException {
+        protected long getTemplateVirtualSize(File file) throws IOException {
```

Comment:
Can you fix the method indentation here.  This should not be indented that extra tab.